docs/ops.html

```diff
@@ -1044,8 +1045,9 @@ <h4 class="anchor-heading"><a id="georeplication-monitoring" class="anchor-link"
 replication-latency-ms-max
 replication-latency-ms-avg
 byte-rate               # average number of bytes/sec in replicated records
+byte-count              # number of bytes replicated source -> target
 
-# MBean: kafka.connect.mirror:type=MirrorCheckpointConnector,source=([-.w]+),target=([-.w]+)
+# MBean: kafka.connect.mirror:type=MirrorCheckpointConnector,source=([-.w]+),target=([-.w]+),topic=([-.w]+),partition=([0-9]+),group=([-.w]+)
```

Comment:
The order to the attribute is source, target, group, topic and partition.

Reply:
Done!